renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "config:base"
```

Comment:
Could we make this the same as the other configurations?

Reply:
surely!  missed this one